addons/website_sale_comparison/static/src/js/website_sale_comparison.js

```diff
@@ -139,6 +153,9 @@ var ProductComparison = Widget.extend({
         this.refresh_panel();
     },
     rm_from_comparelist: function(e){
+        $.when(this.guard).then(this._rm_from_comparelist.bind(this, e));
+    },
+    _rm_from_comparelist: function(e) {
```

Comment:
linter: space after `function`

Reply:
If I can have an opinion here, the rest of the code isn't linted. I think it is better to be consistent with the current file than break consistency inside a module.

https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

Reply:
thank you @Julien00859 , but we prefer now that the code grow quickly, to lint all new code or updated code when you are editing a line that is wrong.

What we don't want is to have big diff just for lint, it is why we don't re-lint all file once a time.

> it is a small step for this Odoo, but a giant step for this file.

Reply:
As long as the linting is fixed at forward port in 12.0 where the file is linted I'm fine with it.